src/providers/twitch/TwitchChannel.cpp

```diff
@@ -754,6 +776,31 @@ void TwitchChannel::refreshChatters()
         .execute();
 }
 
+void TwitchChannel::fetchDisplayName()
+{
+    getHelix()->getUserByName(
+        this->getName(),
+        [weak = weakOf<Channel>(this)](const auto &user) {
+            auto shared = weak.lock();
+            if (!shared)
+                return;
+            auto channel = static_cast<TwitchChannel *>(shared.get());
+            if (QString::compare(user.displayName, channel->getName(),
+                                 Qt::CaseInsensitive) == 0)
+            {
+                channel->setDisplayName(user.displayName);
+                channel->setLocalizedName(user.displayName);
+            }
+            else
+            {
+                channel->setLocalizedName(user.displayName + " (" +
+                                          channel->getName() + ")");
```

Comment:
Could make use of `QString().arg()` in here to make things look more simple. Also, to keep consistency with chat messages, shouldn't we keep format `loginName(localizedName)`?
![](https://cdn.zneix.eu/NGrqZZ8.png)

Reply:
> Could make use of [`QString().arg()`](https://doc.qt.io/qt-5/qstring.html#arg-12) in here to make things look more simple

Sure
> Also, to keep consistency with chat messages, shouldn't we keep format loginName(localizedName)?

Yep, forgot that chatterino formats it swapped.